superset/exceptions.py

```diff
@@ -68,3 +68,7 @@ class CertificateException(SupersetException):
 
 class DatabaseNotFound(SupersetException):
     status = 400
+
+
+class ChartDataValidationError(SupersetException):
```

Comment:
Can we name it `QueryObjectValidationError` or `QueryContextValidationError` and assign a `status = 400`? One may confuse `ChartData` with the response query results, and not realize it is about the request parameters.